File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.kt
class Solution {
	fun solution(phone_number: String): String {
		var answer = ""

		answer = phone_number.substring(0, phone_number.length - 4).replace("[0-9]".toRegex(), "*")
		val combine = phone_number.substring(phone_number.length - 4 , phone_number.length)

		return answer + combine
	}
}

File: 프로그래머스/lv1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Linq;

public class Solution
{
    public string solution(string phone_number)
    {
        StringBuilder sb = new StringBuilder();
        string answer = "";

        char[] numChar = phone_number.ToCharArray();

        for (int i = 0; i < numChar.Length ; i++)
        {
            if (i < numChar.Length-4)
            {
                numChar[i] = '*';
            }
            answer += numChar[i];
        }

        return answer;
    }
}